apps/home/routes.py

```diff
@@ -334,6 +338,10 @@ def edit_lab(lab_id):
     # validate lab category
     # validate mandatory fields
     # ...
+
+    if request.form["lab_category"] not in [str(cat_id) for cat_id in lab_categories]:
```

Comment:
@IsraelPedreira, the lab_category attribute should be an integer, so if it is not an integer (because the way data is send over the request form), you should convert it (make sure to validate it or handle possible conversation exceptions). That way you won't need to loop again over the lap categories just to convert it to string ;)